models.py

```diff
@@ -110,7 +119,14 @@ def generate_compiled_segmentation_model(model_name, model_parameters, num_class
 
     # strategy = tf.distribute.MirroredStrategy()
     # with strategy.scope():
-    model = Unet(input_shape=(None, None, 1), classes=num_classes, **model_parameters)
+    if model_name == "Unet":
+        model = Unet(classes=num_classes, **model_parameters)
+    elif model_name == "FPN":
+        model = FPN(classes=num_classes, **model_parameters)
+    elif model_name == "Linknet":
+        model = Linknet(classes=num_classes, **model_parameters)
+    else:
+        raise NameError("Error, model name not Unet, FPN, or Linknet.")
```

Comment:
Shouldn't we add a few asserts here too based on the notes you have on the excel sheet? e.g. if you know that a given model does not work with a given backbone or something, maybe we can assert this. This will make the next user's life easier since he will know what is breaking and why